process/modules/fin_to_conv.py

```diff
@@ -0,0 +1,121 @@
+from collections import defaultdict
+from typing import List
+import stanza
+import udon2
+from stanza.utils.conll import CoNLL
+import pyconll
+import pymorphy2
+from process.pipeline import ParaphrasePipeline
+from process.module import ParaphraseModule
+import re
+from process.preprocessing_utils import PreprocessingUtils
+
+class FintoConv(ParaphraseModule):
+    def __init__(self, name="fin_to_conv") -> None:
+        super().__init__(name=name)
+        
+    def load(self, preproc_utils: PreprocessingUtils) -> None:
+        # load any tools as `preproc_utils` attributes
```

Comment:
Лучше загрузить morph как атрибут preproc_utils, чтобы не дублировать загрузку:

```suggestion
        if getattr(preproc_utils, 'morph', None) is None:
            preproc_utils.morph = pymorphy2.MorphAnalyzer()
```